### ISSUE TITLE
TypeError on get_files and get_templates

### ISSUE THREAD

**Issue by psychopenguin** [flaper87/pyhelm#57]
It seems the protobuf based objects - Template and Any - expect to receive it's data contents as `bytes`but `string` is being passed, resulting in a TypeError exception when calling `get_files` and `get_templates` methods for a `Chart` object.

Here is an example on how to reproduce:

```
>>> from pyhelm import chartbuilder
>>> chart = {"name": "nginx-ingress", "source": {"type": "repo", "location": "https://kubernetes-charts.storage.googleapis.com"}}
>>> c = chartbuilder.ChartBuilder(chart)
2019-03-27 11:25:43,613 - ChartBuilder - INFO - Cloning https://kubernetescharts.storage.googleapis.com/ for release nginx-ingress
>>> c.get_templates()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/fabio/.local/lib/python3.7/site-packages/pyhelm/chartbuilder.py", line 200, in get_templates
    'r').read()))
TypeError: '{{- if .Values.udp }}\napiVersion: v1\nkind: ConfigMap\nmetadata:\n  labels:\n    app: {{ template  has type str, but expected one of: bytes
>>> c.get_files()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/fabio/.local/lib/python3.7/site-packages/pyhelm/chartbuilder.py", line 152, in get_files
    chart_files.append(Any(type_url=filename, value=fd.read()))
TypeError: '# nginx-ingress\n\n[nginx-ingress](https://github.com/kubernetes/ingress-nginx) is an Ingress contr has type str, but expected one of: bytes
```

**Comment by genti-t** [comment]
Is this fixed?
I see it is merged, but I still see this issue.

**Comment by yanivoliver** [comment]
No, the merge was reverted as it broke Python3 tests.

**Comment by Deepnekroz** [comment]
I also facing this issue. Are there any estimates for a fix to be merged?

**Comment by yanivoliver** [comment]
I've resolved the issue as @psychopenguin's PR did, but updated the tests accordingly. @Deepnekroz, @genti-t - I've released version 0.1.1 with the merged changes - please give it a shot.
